cgi-bin/github_webhook

```diff
@@ -45,6 +45,7 @@ def check_event(data):
   if (event == 'pull_request'): return True,event
   elif (event == 'issues') and (not data['action'] in ['opened']): return False,event
   elif (event == 'issue_comment'):
+    if data["action"] == "edited" and (data['comment']['user']['login'] == "cmsbuild"): return False, event
```

Comment:
@iarspider , instead of hardcoding `cmsbuild` , please use
```
try: cmsbuild_user =  repo_config.CMSBUILD_USER
except: cmsbuild_user = "cmsbuild"
```

For some repository the bot user is `cmsbot`